src/posts/experimental-static-build.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 'Scaling Astro Sites to Thousands of Pages'
```

Comment:
```suggestion
title: 'Scaling Astro's Build Process'
```

Slightly tighter title, totally just IMO.

Reply:
I like the idea of a number (numbers always hit better). If we want more concise, maybe: 

`title: 'Scaling Astro to 10,000+ Pages'`